tarentula/metadata_fields.py

```diff
@@ -1,18 +1,31 @@
 import requests
 from json import dumps
 
+from tarentula.datashare_client import DatashareClient
+from tarentula.logger import logger
 from tarentula.datashare_client import urljoin
 
 
 class MetadataFields:
     def __init__(self,
+                 datashare_url: str = 'http://localhost:8080',
```

Comment:
I see there is a default constant DATASHARE_DEFAULT_PROJECT defined in [datashare_client.py ](https://github.com/ICIJ/datashare-tarentula/blob/8abb06786b271d43850a04e6815ae59cc67127ab/tarentula/datashare_client.py#L15) 
It could be interesting to have a single file handling these constant to ease access, update and prevent typos. Maybe that's another issue